src/bindist/autoexec.d/11temp.bat

```diff
@@ -1,2 +1,6 @@
 @rem Point %TEMP% to a valid directory
-set TEMP=c:\tmp
\ No newline at end of file
+if exist c:\tmp\nul set TEMP=c:\tmp
+if not %TEMP% == "" goto end
+lredir2 t: linux\fs\tmp

```

Comment:
Why is so?
Just mkdir tmp?

Reply:
Either is fine with me. I could also do that and test whether c:\tmp could be created or not (maybe c:\ is readonly or c:\tmp is an existing file) and fall back to /tmp after that.

Reply:
I guess might be useful to keep the lines which just test for %TEMP% being set and if not, redirect it to /tmp. Then the line `set temp=c:\tmp` will move back to autoexec.bat which ensures that the Windows installer will see it. As these scripts are ran at the end of autoexec.bat, only when a user somehow removes the usual line, this fallback will result in a proper %TEMP% even when c:\\ is readonly.
I think there were some FreeDOS installation variants that had (parts of?) c:\\ readonly?